ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+#### Description:
+
+#### Steps to reproduce (if applicable):
+
+#### Helpful external resources (blog, docs etc.):
+
+#### Notes:
+
+_contact_ support@huboard.com _for personal support on critical issues_
```

Comment:
> We could potentially wrap this in an HTML comment if we don't intend for it to be visible. Not sure what you had in mind, I could go either way.

Maybe, not sure how it would play with our meta-data comment.